### PR TITLE
Minor edits related to XSpectrum1D

### PR DESCRIPTION
**Pull request by profxj**
Plotting and I/O

No new tests or docs

### CODE REVIEW

**Comment by profxj** [comment]
Lots of fixes for Python 3.
Now passing tests

File: linetools/spectra/io.py [hunk 1]
```diff
@@ -124,7 +125,7 @@ def readspec(specfil, inflg=None, efil=None, verbose=False, multi_ivar=False,
             if debug:
                 print(
   'linetools.spectra.io.readspec(): Assuming separate flux and err files.')
-            xspec1d = parse_linetools_spectrum_format(hdulist)
+            xspec1d = parse_linetools_spectrum_format(hdulist, **kwargs)
```
Comment (ntejos) [inline linetools/spectra/io.py]: Mention in the docstring of readspec that kwargs are passed down to XSpectrum1D initialization.

Comment (profxj) [inline linetools/spectra/io.py]: added

**Comment by profxj** [comment]
@ntejos please merge

**Pull request merged**